source/CardGames.py

```diff
@@ -196,4 +196,12 @@ def playerTurn(players: list, current_player: str, getCard: bool) -> None:
   else:
       next_player_index = (players.index(current_player) + 1) % num_players
       next_player = players[next_player_index]
-      return next_player
\ No newline at end of file
+      return next_player
+  
+def playerTurn(players, current_player, getCard):
+    num_players = len(players)
+    if getCard:
+        return current_player
+    else:
+        next_player = (current_player + 1) % num_players
+        return next_player 
```

Comment:
Good Job Andre, your code is super clean and concise.

Reply:
Thanks for the review Zach!